quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/DashboardController.java

```diff
@@ -5,40 +5,54 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.github.reflectoring.haljson.HalJsonResource;
+import com.github.reflectoring.infiniboard.packrat.dashboard.DashboardConfig;
+import com.github.reflectoring.infiniboard.packrat.dashboard.DashboardConfigRepository;
+import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfigRepository;
 
 import static org.springframework.http.HttpStatus.NOT_FOUND;
 import static org.springframework.http.HttpStatus.OK;
 import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
 @RestController
 @RequestMapping("/api/dashboards")
 public class DashboardController {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)